spec/factories/conferences.rb

```diff
@@ -1,8 +1,12 @@
 FactoryGirl.define do
   factory :conference do
     name { [FFaker::CheesyLingo.title, 'Conf'].join ' ' }
+    url { FFaker::Internet.http_url }
     starts_on { Time.utc(Date.today.year, 7, 7) }
     ends_on { Time.utc(Date.today.year, 7, 15) }
+    city { FFaker::Address.city }
+    country { FFaker::Address.country }
+    region { 'Africa' }
```

Comment:
(No need to change this, just a Factory Girl hint: you only need to pass a block for dynamic values. The value `'Africa'` is static and can be passed as a normal string argument)